cmake/Modules/CodeCoverage.cmake

```diff
@@ -54,11 +57,20 @@ find_program( LCOV_PATH  NAMES lcov lcov.bat lcov.exe lcov.perl)
 find_program( GENHTML_PATH NAMES genhtml genhtml.perl genhtml.bat )
 find_program( GCOVR_PATH gcovr PATHS ${CMAKE_SOURCE_DIR}/scripts/test)
 find_program( SIMPLE_PYTHON_EXECUTABLE python )
+find_program( PYTHON_COVERAGE_PATH python3-coverage )
```

Comment:
I think we probably want to do this in the reverse direction - default to python-coverage, and then find python3-coverage if python-coverage doesn't exist. For users on Melodic (Ubuntu 18.04), both executables will exist, which will cause us to use python3-coverage to parse the results which were likely generated with python2 (which might work, but could have odd bugs). For users on python3-only systems (20.04 and later), python-coverage will not exist and they will find the correct python3 version.

Reply:
Sure, will update in a minute.